sass/layouts/columns.scss

```diff
@@ -0,0 +1,110 @@
+.tool {
+  @include layout-3-columns();
+  gap: 50px;
+  @include media-1020 {
+    @include layout-3-columns($columns: 1fr 1fr, $rows: 1fr 1fr 1fr);
+    margin-right: 35px;
+  }
+  @include media-768 {
+    @include layout-3-columns($columns: 1fr, $rows: 1fr 1fr 1fr 1fr 1fr 1fr);
+    margin-right: 0;
+    gap: 15px;
+  }
+  @include media-375 {
+    width: 260px;
+  }
+}
+
+.tool__item-1 {
+  @include layout-2-columns();
+  @include media-768 {
+    gap: 10px;
+    width: 300px;
+    justify-self: center;
+  }
+  @include media-375 {
+    gap: 10px;
+    width: 260px;
+  }
+}
+
+.tool__item-2 {
+  @include layout-2-columns();
+  @include media-768 {
+    gap: 10px;
+    width: 300px;
+    justify-self: center;
+  }
+  @include media-375 {
+    gap: 10px;
+    width: 260px;
+  }
+}
+
+.tool__item-3 {
+  @include layout-2-columns();
+  @include media-768 {
+    gap: 10px;
+    width: 300px;
+    justify-self: center;
+  }
+  @include media-375 {
+    gap: 10px;
+    width: 260px;
+  }
+}
+
+.tool__item-4 {
+  @include layout-2-columns();
+  @include media-768 {
+    gap: 10px;
+    width: 300px;
+    justify-self: center;
+  }
+  @include media-375 {
+    gap: 10px;
+    width: 260px;
+  }
+}
+
+.tool__item-5 {
+  @include layout-2-columns();
+  @include media-768 {
+    gap: 10px;
+    width: 300px;
+    justify-self: center;
+  }
+  @include media-375 {
+    gap: 10px;
+    width: 260px;
+  }
+}
+
+.tool__item-6 {
+  @include layout-2-columns();
+  @include media-768 {
+    gap: 10px;
+    width: 300px;
+    justify-self: center;
+  }
+  @include media-375 {
+    gap: 10px;
```

Comment:
Do you need to repeat `gap: 10px;` here? You already have it for 768.

Reply:
DONE!